vendor/github.com/softlayer/softlayer-go/helpers/product/product.go

```diff
@@ -287,3 +287,33 @@ func GetPackageByKeyName(
 
 	return packages[0], nil
 }
+
```

Comment:
For any changes in vendor we need to raise PR to the vendored repo. Once PR gets merged we will vendor the package in our terraform provider repo using <govendor fetch ...>
For this <GetPriceIdByPackageId> raise PR against softlyer-go repo `https://github.com/softlayer/softlayer-go`

Reply:
Okay. Will raise PR there.